src/sql/workbench/contrib/notebook/find/notebookFindModel.ts

```diff
@@ -585,14 +584,59 @@ export class NotebookFindModel extends Disposable implements INotebookFindModel
 	// In markdown links are defined as [Link Text](https://url/of/the/text). when searching for text we shouldn't
 	// look for the values inside the (), below regex replaces that with just the Link Text.
 	cleanMarkdownLinks(cellSrc: string): string {
-		return cellSrc.replace(/(?:__|[*#])|\[(.*?)\]\(.*?\)/gm, '$1');
+		return cellSrc.replace(/(?:__)|\[([\s\S]*?)\]\(.*?\)/gm, '$1');
 	}
 
-	// remove /n's to calculate the line number to locate the correct element
+	// clean up the source w.r.t rendered html elements to locate the correct element
 	cleanUpCellSource(cellValue: string | string[]): string | string[] {
 		let trimmedCellSrc: string[] = [];
+		// regex to get the elements that span multiple lines into one.
+		const newline = new RegExp(/^([-=]+[\\r\\n]*)$/gm);
+		// regex to clean up table seperators ex: |--|----:| since they aren't rendered as a seprate element
+		const tableSeperator = new RegExp(/^(([|]+[ :]*[-]+[ :]*[|]*)+[\\r\\n]*)$/gm);
+		// regex to match a table row -> | rowData | rowData |
+		const tableRows = new RegExp(/^\|(.+\|*)*/gm);
+		// regex for match un-ordered list elements in markdown
+		const listRegex = new RegExp(/^[*+-]+(.+[\r\n]*)/gm);
+		// regex to match block-quotes that begin and end with >>>
+		const blockQuoutesRgx = new RegExp(/^\>(.*)/gm);
```

Comment:
nit: but you have 3 different naming styles for these. `<name>` `<name>Regex` and `<name>Rgx`. Just pick one and stick with it (`<name>Regex` is my preference` )